beacon/light/api/light_api.go

```diff
@@ -151,7 +152,7 @@ func (api *BeaconLightApi) httpGet(path string) ([]byte, error) {
 }
 
 func (api *BeaconLightApi) httpGetf(format string, params ...any) ([]byte, error) {
-	return api.httpGet(fmt.Sprintf(format, params...))
+	return api.httpGet(fmt.Sprintf(format, params...), nil)
```

Comment:
This `httpGetf` method is a bit useless, IMO. It's now used in three places, so they can write e.g 
```
	resp, err := api.httpGetf("/eth/v1/beacon/headers/%s", blockId)
```
instead of 
```
	resp, err := api.httpGet(fmt.Sprintf("/eth/v1/beacon/headers/%s", blockId))
```
I approve of this patch, but I also wouldn't mind dropping that method.

Reply:
I agree, `httpGetf` removed.